src/ca/legacy/pcas/generic/casdef.h

```diff
@@ -119,14 +120,17 @@ class epicsShareClass pvExistReturn {
     pvExistReturn ( pvExistReturnEnum s = pverDoesNotExistHere );
     // directory service server tools will use this (see caNetAddr.h)
     pvExistReturn ( const caNetAddr & );
+    pvExistReturn ( const caNetAddr &, ca_uint16_t );
```

Comment:
If this API change is to be introduce into PCAS in base 3.15, as opposed to the [current home for PCAS](https://github.com/epics-modules/pcas), then some kind of ID macro is necessary.

Reply:
I will be happy to do this if you can point me at an example of what you're looking for.

Reply:
I added

#define HAS_CA_HOST_MINOR_PROTOCOL

in cadef.h and 

#define HAS_CAS_RETURN_MINOR_PROTOCOL

in cadef.h.